sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -141,6 +152,8 @@ private[sbtplugin] object ScalaJSPluginInternal {
         Tags.Tag(s"uses-scalajs-linker-$projectPart-$configPart-$stagePart")
       },
 
+      usesScalaJSLinkerTag in legacyKey := (usesScalaJSLinkerTag in key).value,
```

Comment:
This one is fine because it is essentially a read-only key from the users' point of view.